source/Camera/Basic/CaptureTutorial.md

```diff
@@ -0,0 +1,121 @@
+## Introduction
+
+This tutorial describes how to use Zivid SDK to capture point clouds and 2D images.
+
+1. The fastest way to get your first capture with the Zivid SDK!
+2. How can you configure the settings of the Zivid camera?
+3. More on captures - HDR and 2D
+4. Bonus!
+
+### Prerequisites
+
+You should have installed Zivid SDK and C++ samples. For more details see [Instructions][installation-instructions-url].
+
+## Part 1 - Quick Start
+
+### Connect
+
+First we have to connect to the camera ([go to source][connect-url]).
+```cpp
+auto camera = zivid.connectCamera();
+```
+
+### Capture
+
+Now we can capture a frame. The default capture is a single 3D point cloud ([go to source][capture-url]).
+```cpp
+auto frame = camera.capture();
+```
+
+### Save
+
+We can now save our results. By default the 3D point cloud is saved in Zivid format `.zdf` ([go to source][save-url]).
+```cpp
+auto resultFile = "result.zdf";
+frame.save(resultFile);
+```
+
+## Part 2 - Settings
+
+### Capture Assistant
+It can be difficult to know what settings to configure. Luckily we have the Capture Assistant. This is available in the Zivid SDK to help configure camera settings ([go to source][captureassistant-url]).
+```cpp
+Zivid::CaptureAssistant::SuggestSettingsParameters suggestSettingsParameters(
+    std::chrono::milliseconds{ 1200 }, Zivid::CaptureAssistant::AmbientLightFrequency::none);
+
+std::cout << "Running Capture Assistant with parameters: " << suggestSettingsParameters << std::endl;
+const auto settingsVector{ Zivid::CaptureAssistant::suggestSettings(camera, suggestSettingsParameters) };
+```
+
+These settings can be used in an [HDR capture](#capture-hdr), which we will discuss later.
+
+As opposed to manual configuration of settings, there are only two parameters to consider with Capture Assistant.
+1. **Allowed budget for capture** in number of milliseconds.
```

Comment:
Allowed budget for capture - I think we should be consistent with the terms we use. In Zivid Studio it is Max capture time.